common/src/main/java/com/redhat/cloud/notifications/models/EndpointType.java

```diff
@@ -1,12 +1,17 @@
 package com.redhat.cloud.notifications.models;
 
-import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.annotation.JsonProperty;
 
 public enum EndpointType {
```

Comment:
Should `@Schema` be added back here?

Reply:
Sure, yeah, let's add it back just in case.